### ISSUE TITLE
Display the mana cost exactly as printed

### ISSUE THREAD

**Issue by gaddlord** [gaddlord/mtg.studio#356]
zorbaelbudha:
If you see on the grids the cost of Dauntless Escort is shown as 1WG, but in the card is 1GW, and ALL the cards that have xWG are always shown as xGW on the card. the same is true for the xUG cards, their mana costs on the cards begin with the G instead of the U.

![image](https://user-images.githubusercontent.com/279005/166815132-1694c5c7-4236-4e83-a9d6-aeb23eeede78.png)

It is because for wizards is a color wheel.
So it could be seen as W U B R G W U B R.
And they try to associate the symbols for closeness, so instead of useing WG (which has to skip U R B) they go to the path GW.
The same is true for RW or GU.

It is not a big deal.

**Comment by gaddlord** [comment]
Gad:
This seems reasonable but can not be applied as a rule since WotC (for some strange reasons) print one and the same card with different mana cost presentations. See the list below:

```
Arcades Sabboth   2WWGGUU   Legends
Arcades Sabboth   2WWUUGG   Chronicles

Ayesha Tanaka   UUWW   Legends
Ayesha Tanaka   WWUU   Chronicles

Chromium   2BBUUWW   Legends
Chromium   2WWUUBB   Chronicles

Fiery Justice   RGW   Time Spiral
Fiery Justice   WRG   Ice Age

Jasmine Boreal   3GW   Time Spiral
Jasmine Boreal   3WG   Legends

Johan   3WRG   Chronicles
Johan   3WGR   Legends

Marhault Elsdragon   3GRR   Legends
Marhault Elsdragon   3RRG   Chronicles

Nebuchadnezzar   3UB   Chronicles
Nebuchadnezzar   3BU   Legends

Palladia-Mors   2WWGGRR   Legends
Palladia-Mors   2WWRRGG   Chronicles

Ranger en-Vec   1WG   Anthologies
Ranger en-Vec   1GW   Tempest

Rubinia Soulsinger   2WUG   Chronicles
Rubinia Soulsinger   2UWG   Legends

Sivitri Scarzam   5BU   Legends
Sivitri Scarzam   5UB   Chronicles

Sol'kanar the Swamp King   2UBR   Chronicles
Sol'kanar the Swamp King   2RBU   Legends
Sol'kanar the Swamp King   2UBR   Time Spiral
Sol'kanar the Swamp King   2RBU   Cards 6x9

Stangg   4RG   Chronicles
Stangg   4GR   Legends

Tobias Andrion   3WU   Chronicles
Tobias Andrion   3UW   Legends

Xira Arien   GRB   Legends
Xira Arien   BRG   Chronicles
```

The correct solution is to display the mana cost symbols as they are typed in the top right corner. It requires some major internal changes in mana cost parser.

**Comment by gaddlord** [comment]
Resolved in version 4.0 since mana is now parsed token by token.
